src/docx/assets/indywidualne/word/fontTable.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version='1.0' encoding='UTF-8' standalone='yes'?>
+<w:fonts mc:Ignorable="w14" xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" xmlns:r="http://schemas.openxmlformats.org/officeDocument/2006/relationships" xmlns:w="http://schemas.openxmlformats.org/wordprocessingml/2006/main" xmlns:w14="http://schemas.microsoft.com/office/word/2010/wordml">
+  <w:font w:name="Symbol">
+    <w:panose1 w:val="00000000000000000000"/>
+    <w:charset w:val="02"/>
+    <w:family w:val="auto"/>
+    <w:pitch w:val="variable"/>
+    <w:sig w:csb0="80000000" w:csb1="00000000" w:usb0="00000000" w:usb1="10000000" w:usb2="00000000" w:usb3="00000000"/>
```

Comment:
I think we can remove all fonts except `TimesNewRoman` and `Symbols` (maybe)

Reply:
Agree, but not in that pull request. I can create separate issues for:
 * `docProps/core.xml` to contain `alinka.io` as a generator and proper time of generation
 * refine set of fonts that we, in fact, going to use in our rendered documents.

This pull request is only about proving that using templates should be an efficient way to generate documents based on provided values.